tests/ConfigurationLoaderTest.hack

```diff
@@ -15,17 +15,17 @@ use function Facebook\FBExpect\expect;
 final class ConfigurationLoaderTest extends \Facebook\HackTest\HackTest {
   const IGNORED_VALUE = '__ignore__';
 
-  public function goodTestCases(): array<string, array<array<string, mixed>>> {
-    return darray[
-      'fully specified' => varray[darray[
+  public function goodTestCases(): dict<string, (darray<string, mixed>)> {
```

Comment:
Mixing these feels /really/ weird; this should really be one out of all darray, all dict, or dict<string, shape(.......)>>

Reply:
I agree, this was an oversight in my programming.
I found this out later while coding on lexidor#1, where they are all dicts.